frontend/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:12-alpine as builder
+
+WORKDIR /usr/src/app
+
+COPY ./frontend ./frontend
```

Comment:
COPY package*.json ./

Reply:
Done

Reply:
check buildkit option for docker - it allows to use mounts instead of copy. Mixing it with multistage build it will improve build time significantly.

Reply:
Docker build action which we use does not support BUILDKIT - https://github.com/whoan/docker-build-with-cache-action#cache-is-not-working